src/w3d/lib/multilist.h

```diff
@@ -19,6 +19,8 @@
 #include "autopool.h"
 #include <captainslog.h>
 
+// #TODO investigate C casts in this file.
```

Comment:
is this note still valid?

Reply:
Yes. There is a lot going on in this file and it's too much to unpack easily.